authors/apps/comments/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+
+from .views import (
+    CommentDetailsAPIView, CommentsAPIView, ReplyAPIView, ReplyDetailsAPIView
+)
+urlpatterns = [
+    path('<slug>/comments/', CommentsAPIView.as_view(), name='comments'),
+    path('comments/<pk>', CommentDetailsAPIView.as_view(),
```

Comment:
@sebbss , the comment URLs are conflicting. The routes should be matching with the article slug ie 
Lets have the following 
- creating and viewing all comments on an article  `<slug>/comments/`
- editing and deleting  a comment  `<slug>/comments/<pk>`
- creating a reply and viewing replies on a comment `<slug>/comments/<pk>/replies/`
- viewing, editing and deleting a single reply `<slug>/comments/<pk>/replies/<pk>`
Also ensure that only the user who made a comment can delete it.